doc/api/packages.md

```diff
@@ -252,8 +252,6 @@ absolute subpath of the package such as
 
 ### Subpath exports
 
-> Stability: 1 - Experimental
-
```

Comment:
Should these get YAML with version added/version stable etc.?

Reply:
The existing stable features do not have stability indexes so this seemed consistent